src/include/sof/audio/volume.h

```diff
@@ -66,7 +66,7 @@ struct sof_ipc_ctrl_value_chan;
 #define COMP_VOLUME_Q1_23 1
 
 //** \brief Volume gain Qx.y integer x number of bits including sign bit. */
-#define VOL_QXY_X 1
+#define VOL_QXY_X 8
```

Comment:
Please add each dynamic range before and after your convert, in theory, if input is Q1.31, why output need Q8.23 or Q2.23?

Reply:
Yep, I'll add to commit text and code comment to header. With Q8.23 the gain can be up to 42.144 dB.  Lowest non-zero gain is -138.47 dB.

Reply:
done!